### DISCUSSION TITLE
Support for Google reCAPTCHA v3

### DISCUSSION THREAD

**Discussion by DotCat1985** [OrchardCMS/OrchardCore#11520] [General]
Hi,
when will **OrchardCore.ReCaptcha** module support for **Google reCAPTCHA v3**?

**Comment by Skrypt** [comment]
When someone kind enough will work on it. 😄 

Nothing is planned. V3 seems to need less integration than V2. It is harder to manage the threshold of each form individually too. V2 is somewhat practical for that matter.
